packages/node-client/lib/index.ts

```diff
@@ -94,31 +94,65 @@ export class Nango {
      * =======
      */
 
+    /**
+     * Returns a list of integrations by sending a GET request
+     * @returns {Promise<{ configs: Pick<Integration, 'unique_key' | 'provider'>[] }>} A promise that resolves with an object containing an array of integration configurations
+     */
     public async listIntegrations(): Promise<{ configs: Pick<Integration, 'unique_key' | 'provider'>[] }> {
         const url = `${this.serverUrl}/config`;
         const response = await axios.get(url, { headers: this.enrichHeaders({}) });
 
         return response.data;
     }
 
-    public async getIntegration(providerConfigKey: string, includeIntegrationCredentials = false): Promise<{ config: Integration | IntegrationWithCreds }> {
+    /**
+     * Returns a specific integration by sending a GET request
+     * @param {string} providerConfigKey - The key identifying the provider configuration on Nango
+     * @param {boolean} [includeIntegrationCredentials=false] - Whether to include integration credentials in the response. Default is false
+     * @returns {Promise<{ config: Integration | IntegrationWithCreds }>} A promise that resolves with an object containing an integration configuration
+     */
+    public async getIntegration(
```

Comment:
This comment for all jsDoc, I don't think it's necessary to put the type again imo.
For example here:
```ts
    /**
     * Returns a specific integration by sending a GET request
     * 
     * @param providerConfigKey - The key identifying the provider configuration on Nango
     * @param includeIntegrationCredentials - Whether to include integration credentials in the response. Default is false
     * @returns A promise that resolves with an object containing an integration configuration
     */
```

Since we also ship the d.ts Typescript does show the actual types and I fear it will not be maintained correctly

<img width="866" alt="Screenshot 2024-04-12 at 10 24 16" src="https://github.com/NangoHQ/nango/assets/1637651/c9242d23-5c81-4465-9ee8-33acd7295adc">